src/svcutilcore/pkg/files/svcutilcore.targets

```diff
@@ -0,0 +1,29 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+ <PropertyGroup>
+   <MyProperty>$(IntermediateOutputPath)</MyProperty>
+    <_SerializationAssemblyName>$(AssemblyName).XmlSerializers</_SerializationAssemblyName>
+    <_SerializerDllIntermediateFolder>$(IntermediateOutputPath)$(_SerializationAssemblyName).dll</_SerializerDllIntermediateFolder>
+    <_SerializerPdbIntermediateFolder>$(IntermediateOutputPath)$(_SerializationAssemblyName).pdb</_SerializerPdbIntermediateFolder>
+    <_SerializerCsIntermediateFolder>$(IntermediateOutputPath)$(_SerializationAssemblyName).cs</_SerializerCsIntermediateFolder>
+    <_SvcUtilWarningText>Warning : Fail to generate the serializer for $(AssemblyName)$(TargetExt).</_SvcUtilWarningText>
+    <_SerializationAssemblyDisabledWarnings>$(NoWarn);219;162;$(SerializationAssemblyDisabledWarnings)</_SerializationAssemblyDisabledWarnings>
+  </PropertyGroup>
+
+  <Target Name="SvcUtilGenerateSerializationAssembly" AfterTargets="Build">
+	  <Message Text="Path3 = $(IntermediateOutputPath)" Importance="high" />
```

Comment:
NIT: please fix the indentation.

BTW, why Path**3**?

Reply:
That's used for debug purpose. Will delete it.